Sources/FoundationEssentials/FileManager/FileManager+Utilities.swift

```diff
@@ -28,8 +28,20 @@ import Glibc
 internal import _CShims
 #elseif os(Windows)
 import CRT
+import WinSDK
 #endif
 
+#if os(Windows)
+extension FILETIME {
+    package var timeIntervalSince1970: TimeInterval {
+        var count: Double = Double((UInt64(self.dwHighDateTime) << 32) | UInt64(self.dwLowDateTime))
+        count /= 1e7 // 100 nanoseconds to seconds
+        return count - Date.timeIntervalBetween1601AndReferenceDate
```

Comment:
Thanks for updating this, I like the `timeIntervalSince1970` spelling a lot better (and it mirrors the `Date` API nicely). I think this is almost correct: if `count` is the interval between 1601 and the time in question, then subtracting `timeIntervalBetween1601AndReferenceDate` gets us the interval between the reference date (2001) and the time in question. I think we need to add `Date.timeIntervalBetween1970AndReferenceDate` here if we want this to be 1970-aligned (or we should rename this to `timeIntervalSinceReferenceDate` and use that `Date` initializer instead of we want this to be 2001-aligned)

Reply:
Ah, okay - that does end up being exactly what I had initially - except I don't remember why we ended up reworking this 🤔

Reply:
FWIW, I find it to be a bit verbose of a spelling and not very helpful, but 🤷‍♂️

Reply:
Cool thanks for updating. Yeah it's a bit more verbose, but I think it'll be quite helpful for other developers reading this since it makes it very clear whether you should use this with `Date(timeIntervalSince1970:)` vs. `Date(timeIntervalSinceReferenceDate:)` for example. But this all looks good to me now